bucket/kubectl.json

```
@@ -1,5 +1,6 @@
 {
     "homepage": "https://kubernetes.io/docs/user-guide/kubectl-overview/",
+    "description": "A command-line tool for running commands against Kubernetes clusters.",
```

https://kubernetes.io/docs/reference/kubectl/kubectl/

```suggestion
    "description": "Control the Kubernetes cluster manager.",
```
or

```suggestion
    "description": "Kubernetes cluster manager controler.",
```